docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -84,7 +84,17 @@ This will update the ECK installation to the latest binary and update the CRDs a
 [id="{p}-beta-to-ga-rolling-restart"]
 === Control rolling restarts during the upgrade
 
-Upgrading the operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods by Kubernetes to apply those changes. If you have a very large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart might be disruptive or inconvenient. To have more control over when the pods belonging to a particular deployment should be restarted, you can <<{p}-exclude-resource,add an annotation>> to the corresponding Elasticsearch, Kibana, or APM Server resources to temporarily exclude them from being managed by the operator. When the time is convenient, you can remove the annotation and let the rolling restart go through.
+Upgrading the operator results in a one-time update to existing managed resources in the cluster. This potentially triggers a rolling restart of pods by Kubernetes to apply those changes. The following table shows the target version that would cause a rolling restart.
+
+[float]
+[id="{p}-version-upgrades-causing-rolling-restarts"]
+===== Target version upgrades causing rolling restarts
+[cols="3"]
+|===
+|1.6|1.7|1.9
+|===
```

Comment:
![image](https://user-images.githubusercontent.com/976373/140273913-54a1d7af-2b26-4cfe-979a-71c010cfba6a.png)

An alternative would be to just replace the table with a list:

```suggestion
Upgrading the operator results in a one-time update to existing managed resources in the cluster. This potentially triggers a rolling restart of pods by Kubernetes to apply those changes. Upgrading the operator to the following versions may cause a rolling restart:

* 1.6
* 1.7
* 1.9
```

_it would require to replace `{p}-version-upgrades-causing-rolling-restarts` where it is used with `{p}-beta-to-ga-rolling-restart`_

Reply:
👍  the table is now a list, and the links to this section have been updated.